tests/auto/auto_expts_list.txt

```diff
@@ -0,0 +1,2 @@
+grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GSD_SAR
```

Comment:
Is this file used at this point?

Reply:
You're right, that file is not used now.

Reply:
You're right, it's not used at this point.